pkg/transforms/policyreport.go

```diff
@@ -117,8 +124,11 @@ func PolicyReportResourceBuilder(pr *PolicyReport) *PolicyReportResource {
 	policyList := policies.UnsortedList()
 	sort.Strings(policyList)
 
-	// "rules" is incorrect since there is a "rule" field in the results, but this is kept for backwards compatibility
-	node.Properties["rules"] = policyList
+	ruleList := rules.UnsortedList()
+	sort.Strings(ruleList)
+
+	node.Properties["ruleNames"] = ruleList
+	node.Properties["policies"] = policyList
```

Comment:
1. Let's keep the name `rules`, based on the discussion in previous comment.
2. Please create a separate branch and PR with this change, everything in this file goes together.

I'm good to merge after these changes.

Reply:
After this PR merge, I will rebase https://github.com/stolostron/search-collector/pull/365 PR